components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/util/designview/beans/configs/siddhielements/sourcesink/SourceSinkConfig.java

```diff
@@ -64,4 +65,14 @@ public String getType() {
     public MapperConfig getMap() {
         return map;
     }
+
+    public boolean isCorrelateIdExist() {
+
```

Comment:
Follow the standard across the code. Either 1st line of the method should be a line space but it seems there are some methods without any line space. As a best practice avoid have a line space as the 1st line in a method.

Reply:
fixed